### ISSUE TITLE
[MTOA-34] USD Camera prim bug with Arnold

### ISSUE THREAD

**Issue by jcoelho88** [Autodesk/maya-usd#1815]
Hello,

I recently updated the plugins we are testing with and have noticed that the Arnold Render View does not seem to recognize the USD camera prims, they default to something called "defaultUfeProxyCameraShape" and if you try to render through it you will get this error:

```
// Error: [mtoa] Setting camera defaultUfeProxyCameraShape failed // 
```

Here is what I'm using:
- Windows 10
- Maya 2022.2
- Arnold MtoA 5.0.0.1
- Maya-usd 0.13.0

Steps to reproduce:
- Create a stage with a new layer.
- Create a Camera prim (see pictured)
- Open the Arnold render view and try to render (you will notice the USD camera prim was renamed)
- Check script editor for error

In earlier version of the plugin I think 0.11.0 or 0.12.0 this was not an issue that I encountered.

The expected behavior is for the scene to render from the USD camera prim. If there is some kind of incorrect configuration of plugins please let me know. If this is a known issue, is there a way around this? Should I wait for a fix or possibly revert to a previous version?

![image](https://user-images.githubusercontent.com/88895104/140810182-d959f1e2-1421-4357-990c-ba67345a32b2.png)

Any assistance with this would be greatly appreciated.

Cheers,

Jason Coelho

**Comment by williamkrick** [comment]
Hi @jcoelho88,

defaultUfeProxyCameraShape is the name of the internal camera node Maya uses to represent a Ufe camera in Maya. MayaUSD implements the Ufe interface so that USD camera's can work in Maya.

mtoa doesn't have support for this special camera yet, which is why you are seeing this error. To render in Arnold you still have to import the camera into Maya and render with the corresponding Maya camera.

**Comment by jcoelho88** [comment]
Hi @williamkrick,

Thanks for the clarification, I'll have to see about a change in our workflow until mtoa will recognize the Camera prim. 

With importing the camera, as far as I can tell If I want to get the results of the composed layers of the camera from the stage I would need to write out a new specific layer then import that into the Maya scene. I'll test this out.

Should this issue remain open?

Cheers,

Jason Coelho

**Comment by williamkrick** [comment]
I think we should keep this one open as work for mtoa to support defualtUfeProxyCameraShape.

**Comment by swillisart** [comment]
Just chiming in here that the USD camera is not available in the Renderable Cameras in Maya's Render settings unless you bring in to edit or export.

![22-06-10_05-13-25_c](https://user-images.githubusercontent.com/33715288/172994962-1d11ff2b-b9a8-42ac-8b35-33a7d900f301.gif)
I hope we can get renderable cameras soon.

**Comment by sebastienblor** [comment]
Hi @swillisart , I wanted to mention that it's now possible to render USD cameras with Arnold, without having to first display them in the viewport. They should all be listed in the renderview, and also show up in the renderable cameras list, assuming the camera primitive purpose is either Render or Default.

This is included in the latest MtoA release 5.4.1.0

**Comment by santosg87** [comment]
closing this issue based on the last 2 comments. :)
